lib/kv-extreme-scale.js

```diff
@@ -2,9 +2,12 @@
 // Node module: loopback-connector-kv-extreme-scale
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-
```

Comment:
I personally prefer to keep an empty line after the license boilerplate, I believe other files are doing the same, so it's better to preserve consistency.